config/routes.rb

```diff
@@ -7,7 +7,7 @@
     get "/about", to: "static_pages#about"
     get "/contact", to: "static_pages#contact"
     get "/signup", to: "users#new"
-    # user pages
-    get "users/new"
+    # resources
+    resources :users
```

Comment:
chỉ define cho actions nào đang dùng thôi em
```suggestion
    resources :users, only: %i(new create ...)
```